README.md

```diff
@@ -40,7 +37,7 @@ And for the height data here: [Here](https://github.com/Provincie-Zuid-Holland/v
 
 ## Input normalization/scaling.
 
-For all the models we have used we have normalized/scaled the RGBIH values between 0 and 1.
+Decisions can be made to normalized/scaled the RGBIH values between 0 and 1.
```

Comment:
We primarily use Standard Scaler now, which doesn't actually normalize between 0 and 1, but scales the data to a Standard Distribution of mean 0 and variance 1 (or as close as it can get given the data).

Reply:
In the end i did not do this for Coepelduynen

Reply:
Okay dokay!